### ISSUE TITLE
Wildcard symbol doesn't play well with fish shell

### ISSUE THREAD

**Issue by slifin** [dotnet/templating#482]
### Description

```
~/C/myfableapp  dotnet new -i Fable.Template::*
No matches for wildcard 'Fable.Template::*'.  (Tip: empty matches are allowed in 'set', 'count', 'for'.)
fish: dotnet new -i Fable.Template::*
```

A work around is to step into bash from fish and then execute the command there seems to work ok

originally reported here: https://github.com/fable-compiler/Fable/issues/768

**Comment by sayedihashimi** [comment]
Try dotnet new -i "Fable.Template::*"

**Comment by mlorbetske** [comment]
This is "fixed" in #526 - you no longer need "::*" to be appended in order to just install latest. If you do need to include a star (ex. you want to install the latest in a particular major version, but not the overall latest), you'll need to escape it as appropriate for your shell.
